src/config/configuration.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I know that JSON is more native for Typescript/JS. But what do you think about using YAML as the configuration format? Some benefits are:
1. More human friendly
2. The possibility of adding a comment, mainly since we use some abbreviation or terms that are probably not so obvious after we don't touch it for 6 months.

Reply:
In the second version of solar all configuration values ​​can be modified in a backoffice. I chose the json format because in my opinion it will be the easiest to handle for the rest. But you're right, the .yaml format has other advantages. In any case, it will be necessary to resume this configuration file in the second version and perhaps even put all this information in a database. It is therefore best to resume the format if necessary during the second version of solar.